src/home_robot/home_robot/mapping/semantic/instance_tracking_modules.py

```diff
@@ -739,6 +745,32 @@ def add_view_to_instance(
                 global_instance_id,
             )
 
+    def _interpolate_image(
```

Comment:
Should this be in utils/image.py instead?

Reply:
Done